GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -633,81 +708,52 @@ extension CameraViewController {
         }
     }
     
-    fileprivate func validate(importedDocuments documents: [GiniVisionDocument]) {
+    fileprivate func validate(importedDocuments documents: [GiniVisionDocument],
+                              completion: @escaping ([GiniVisionDocument]) -> Void) {
         let loadingView = addValidationLoadingView()
```

Comment:
You are working with views in a `validate` method??? Not good!

Reply:
It was there from previous implementation where it made sense, but now as you say, it is not good! 👍